packages/hash/integration/src/mockData/index.ts

```diff
@@ -69,30 +70,43 @@ void (async () => {
   const results = new Map<string, Entity>();
 
   const requiredBlockTypes = [
-    "Divider",
-    "Embed",
-    "Image",
-    "Table",
-    "Code",
-    "Video",
-    "Header",
+    ["Divider", "https://blockprotocol.org/blocks/@hash/divider"],
+    ["Embed", "https://blockprotocol.org/blocks/@hash/embed"],
+    ["Image", "https://blockprotocol.org/blocks/@hash/image"],
+    ["Table", "https://blockprotocol.org/blocks/@hash/table"],
+    ["Code", "https://blockprotocol.org/blocks/@hash/code"],
+    ["Video", "https://blockprotocol.org/blocks/@hash/video"],
+    ["Header", "https://blockprotocol.org/blocks/@hash/header"],
   ] as const;
 
   const requiredOtherTypes = ["Company", "Location", "Person"] as const;
   // create the types we'll need below so we can assign their ids to entities
   const newTypeIds: Record<
-    typeof requiredBlockTypes[number] | typeof requiredOtherTypes[number],
+    typeof requiredBlockTypes[number][0] | typeof requiredOtherTypes[number],
     string
   > = {} as any;
 
+  const componentIdSchema = async (componentId: string) => {
+    const blockSchema = await (await fetch(`${componentId}/block-schema.json`))
+      .json()
+      .catch(() => {
+        logger.error(
+          `Could not fetch block-schema.json for componentId ${componentId}. Defaulting to empty schema.`,
+        );
+        return {};
+      });
+
+    return { componentId, ...(blockSchema as object) };
+  };
```

Comment:
Is it a problem that this is basically duplicated from `entity.model.ts`?

Reply:
Yeah, I think that was an oversight, I'm sharing the function between the two now :)